1208-projects/user_edits_project.feature

```diff
@@ -0,0 +1,93 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User with access rights for Project
+        When they open the Project Wizard
+        Then they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+
+    @xxx
+    Scenario: User opens a Project in the Project Wizard
+        Given User with access rights for Project
```

Comment:
Som nevnt i forrige kommentar må vi nok ha noe bedre scenario for dette med hvem som skal ha tilgang til ja (ala `Project Manager opens Landing Page for Project`). Når jeg leser dette vet jeg fortsatt ikke hvem som skal kunne gjøre dette (altså hvem `User with access rights for Project` er)

Reply:
Har skrevet om scenario